pyproject.toml

```diff
@@ -2,11 +2,12 @@
 [project]
 name = "foxglove-sdk-monorepo"
 version = "0.0.0"
-requires-python = ">=3.9"
+requires-python = ">=3.11"
```

Comment:
Python 3.9 is end of support Oct 2025, 3.10 Oct 2026.

I think this fine to require 3.11 as a minimum.

Reply:
I assume we had kept it older intentionally, but sphinx 8.x required at least 3.10 and 8.2 3.11 so 🤷

Reply:
We didn't have a good reason for supporting 3.9 other than we could at the time without big sacrifices. 3.11 is 2.5 years old now. I think it's fine to require that.

Reply:
8.x was latest/default when installing. I wasn't familiar with the poetry workspace setup, we can use the already specified one.